server/api/cvrs.py

```diff
@@ -1035,19 +1034,31 @@ def parse_hart_cvrs(
 
     cvr_zip_files: Dict[str, BinaryIO] = {}  # { file_name: file }
     scanned_ballot_information_files: List[BinaryIO] = []
+    hasNonCsvZipFiles = []
     for file_name in file_names:
         if file_name.lower().endswith(".zip"):
             # pylint: disable=consider-using-with
             cvr_zip_files[file_name] = open(
                 os.path.join(working_directory, file_name), "rb"
             )
-        if file_name.lower().endswith(".csv"):
+        elif file_name.lower().endswith(".csv"):
             scanned_ballot_information_files.append(
                 # pylint: disable=consider-using-with
                 open(os.path.join(working_directory, file_name), "rb")
             )
+        else:
+            hasNonCsvZipFiles.append(file_name)
 
-    assert len(cvr_zip_files) != 0  # Validated during file upload
+    # If there are no zip files inside the "wrapper" we assume it was not a wrapper and there was only one cvr zip file uploaded, unwrapped.
+    if len(cvr_zip_files) == 0 and len(scanned_ballot_information_files) == 0:
+        cvr_zip_files[jurisdiction.cvr_file.name] = retrieve_file(
+            jurisdiction.cvr_file.storage_path
+        )
```

Comment:
Do we need to download the file again in this case, or is there a way to reuse it since we already have it?

Reply:
since the wrapper zip is immediately closed I didn't see a obvious way and wanted to err on the side of touching this code as little as possible haha

Reply:
Maybe we could just do `wrapper_zip.seek(0)` and move the closing of the file to later?

Reply:
filing a followup for this https://github.com/votingworks/arlo/issues/2025